skyportal/handlers/api/tag.py

```diff
@@ -51,13 +23,24 @@ def post(self):
         if not tag_text or not isinstance(tag_text, str):
             return self.error("`tag_name` must be provided as a non-empty string")
 
+        if not re.fullmatch(r"[A-Za-z0-9]+", tag_text):
+            return self.error(
+                "`tag_name` must contain only letters and numbers (no spaces, underscores, or special characters)",
```

Comment:
I see the parameter is called `tag_name`, but in your code the variable is called `tag_text`. I believe that a comment left on the previous PR was that having `tag_...` as a naming scheme in the model was not the best since well it's on a tag-related table so we know it's a tag, we don't need that in the column names.

I don't think you changed that in the model or here, could you look into it? Just calling everything `text` seems more consistent and would make most sense for the data model.